packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Deposit/Confirm/template.success.tsx

```diff
@@ -47,6 +47,18 @@ const Success = (props: Props) => {
     ? (intervalToDuration({ end: props.withdrawLockCheck?.lockTime || 0, start: 0 }).days as number)
     : 0
 
+  const targetCoinName = useMemo(() => {
+    const currency = props.defaultMethod?.currency
+
+    if (!currency) return currency
```

Comment:
If there is no currency you will be returning undefined or null. Is this expected? 
If it is then you dont need to add to the return the non-existing value. You could just return null for code clarity